### PR TITLE
 Add kafka config

### PR DESCRIPTION
**Pull request by xuanyuz**

### CODE REVIEW

**Comment by xuanyuz** [comment]
@xupeilin

File: kafka_sender.go [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+	"encoding/json"
+	"github.com/Shopify/sarama"
+	log "github.com/Sirupsen/logrus"
+	sjson "github.com/bitly/go-simplejson"
+	"sync"
+	"time"
+)
+
+type KafkaConfig struct {
+	Hosts []string `json:"Hosts"`
```
Comment (xupeilin) [inline kafka_sender.go]: brokers

File: kafka_sender.go [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		log.Infof("[Send] fields Marshal err:%v", err)
+		return
+	}
+	msg.Value = sarama.ByteEncoder(value)
+	paritition, offset, err := p.producer.SendMessage(msg)
+	if err != nil {
+		log.Infof("Send Message Fail")
+	}
+
+	log.Infof("[Send]Partion = %d, offset = %d, value = %v \n", paritition, offset, fields)
```
Comment (xupeilin) [inline kafka_sender.go]: Debugf

File: log_task.go [hunk 3]
```diff
@@ -31,12 +31,24 @@ func (p *LogTask) AddPeckTask(task *PeckTask) error {
...
+		}
+	} else {
+		p.peckTasks[task.Config.Name] = task
+	}
 	return nil
 }
 
 func (p *LogTask) RemovePeckTask(config *PeckTaskConfig) error {
+	if !p.peckTasks[config.Name].IsStop() {
+		p.peckTasks[config.Name].Stop()
```
Comment (xupeilin) [inline log_task.go]: return error

**Pull request merged**